docs/contracts/inheritance.rst

```diff
@@ -250,6 +252,20 @@ overridden when used with multiple inheritance:
     // No explicit override required
     contract D is B, C {}
 
+More formally, it is not required to override a function (directly or
+indirectly) inherited from multiple bases if there is a base contract
+that is part of all override paths for the signature, and (1) that
+base implements the function and no paths from the current contract
+to the base mentions a function with that signature or (2) that base
+does not implement the function and there is at most one mention of
```

Comment:
Actually:
```suggestion
mentions the function, but does not implement it, and there is at most one mention of
```
If the function isn't even mentioned in the base, it doesn't qualify.